test/cpp/ios/RemoteTestClientCpp/RemoteTestCpp.podspec

```diff
@@ -15,78 +15,39 @@ Pod::Spec.new do |s|
   s.dependency "Protobuf-C++"
   s.dependency "gRPC-C++"
   s.source_files = "src/proto/grpc/testing/*.pb.{h,cc}"
-  s.header_mappings_dir = "RemoteTestCpp"
+  s.header_mappings_dir = "."
   s.requires_arc = false
 
   repo_root = '../../../..'
-  config = ENV['CONFIG'] || 'opt'
-  bin_dir = "#{repo_root}/bins/#{config}"
+  bazel_exec_root = "#{repo_root}/bazel-out/darwin-fastbuild/bin"
```

Comment:
What happens if we didn't build `protoc` using bazel (do we need to handle the case where protoc is not found in `{bin_dir}/protobuf` and default to using the system protoc?

Reply:
> What happens if we didn't build `protoc` using bazel

The ProtoCompiler pod does that.

> do we need to handle the case where protoc is not found in {bin_dir}/protobuf and default to using the system protoc?

For that case, I think that might not actually be desired? The reason we did that check was because `make protoc` does not make one as `bins/protoc` if there is one installed in the system. That is not the case for Bazel.

One small caveat would be: if anyone runs `bazel clean` (or `make clean` in the old version), `protoc`, `grpc_objective_c_plugin`, and `grpc_cpp_plugin` will be deleted. However, cocoapods doesn't know that. It is a common enough case to check if these executables exist before running the command?

Reply:
> It is a common enough case to check if these executables exist before running the command?

A "check" would be unnecessary because if it does not exist, it just fails out with quite understandable message that the executable does not exist. But we may consider if "check + build" is a good approach.